.kodiak.toml

```
@@ -0,0 +1 @@
+version = 1
```

```suggestion
version = 1
merge.method = "squash"
```

I normally merge via `squash` to keep the commit history clean (aka so all intermediate commits were known good states of the repository rather than incremental parts of a PR), but I am pretty sure that @e-dard  prefers the opposite (keeping the commits from PRs when merged to main)

---

(this I think is what @tustvold  is saying above, and as he mentions we can change the method on a per-PR basis so feel free to ignore this suggestion -- of course, I would suggest we default to *MY* preferred method, but that is me being selfish)

---

I don't mind the commits, but the potentially many "sync to main" merge commits are a nuisance; that's what the rebase option is for (preserves commits but offers a linear history). However since the tests are guaranteed to be actually run at merge points, when rebasing there there is the risk of utterly breaking git bisect.

My personal sweet spot is a squashed git history + small PRs + a powerful code review tool where all review history.